httpcore5/src/main/java/org/apache/hc/core5/http/io/entity/EntityUtils.java

```diff
@@ -103,16 +118,14 @@ public static void consume(final HttpEntity entity) throws IOException {
      */
     public static byte[] toByteArray(final HttpEntity entity) throws IOException {
         Args.notNull(entity, "Entity");
+        int contentLength = (int) Args.checkContentLength(entity);
+        contentLength = toContentLength(contentLength);
```

Comment:
@garydgregory Do you mind folding these two one in order to make `contentLength` final? `final int contentLength = toContentLength((int) Args.checkContentLength(entity))` and then everything is going to be fine.

Reply:
@ok2c Done!